From: Johannes Kauffmann <johanneskauffmann@hotmail.com>
Date: Sat, 3 Jun 2023 17:35:09 +0200
Subject: taglib: remove VLC path define check

It is no longer needed with contribs taglib updated to 1.13.

(cherry picked from commit c5851fd495ed6de1d3712e64a2b1cc1e4e9059f4)
---
 modules/meta_engine/taglib.cpp | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/modules/meta_engine/taglib.cpp b/modules/meta_engine/taglib.cpp
index 2891e06..487a152 100644
--- a/modules/meta_engine/taglib.cpp
+++ b/modules/meta_engine/taglib.cpp
@@ -101,8 +101,7 @@ using namespace TagLib;
 #include <algorithm>
 #include <limits>
 
-#if defined(VLC_PATCHED_TAGLIB_IOSTREAM_RESOLVERS) || \
-    TAGLIB_VERSION >= VERSION_INT(1, 13, 0)
+#if TAGLIB_VERSION >= VERSION_INT(1, 13, 0)
 #define USE_IOSTREAM_RESOLVER 1
 #endif
 
